extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -221,6 +227,15 @@ export class AmpLightboxGallery extends AMP.BaseElement {
     const clonedNode = element.cloneNode(deepClone);
     clonedNode.removeAttribute('on');
     clonedNode.removeAttribute('id');
+
+    const ampClasses = [];
+    for (let i = 0; i < clonedNode.classList.length; i++) {
+      const cssClass = clonedNode.classList[i];
+      if (AMP_CSS_RE.test(cssClass)) {
+        ampClasses.push(cssClass);
+      }
+    }
+    clonedNode.classList.remove.apply(clonedNode.classList, ampClasses);
```

Comment:
I think we can remove all classes from the clone. I don't see a lot of situations where the style for the opener should apply to the lightboxed version. If anything it can cause weird bugs.

Reply:
what does amp-image-lightbox do?

Reply:
It only propagates the `aria-attributes`, not the source. 

```
Object.keys(this.ariaAttributes_).forEach(key => {
      this.ariaAttributes_[key] = sourceElement.getAttribute(key);
      if (this.ariaAttributes_[key]) {
        this.image_.setAttribute(key, this.ariaAttributes_[key]);
      }
    });
```